### ISSUE TITLE
limit displayed rows on table

### ISSUE THREAD

**Issue by cartoloupe** [cartoloupe/drift#32]
the table can grow and take up the whole page.
display only the latest N rows of the table, 
with the option to scroll within a window,
such that the height of the table will remain the same
and be in the same place on the page

**Comment by degoeym** [comment]
To be clear, you're not talking about paging the results, you're talking about displaying, say, the first 20 rows and then the user can scroll the table to view the remaining rows?

**Comment by cartoloupe** [comment]
Yep, something like that - they wouldn't have to hit links to page 2 or 3 etc..., but could just scroll down. I'm not opposed to pagination if it is easier though; I'm using the app locally and have accumulated rows beyond the size of the window size.
